app/exporters/patients_exporter.rb

```diff
@@ -16,10 +30,10 @@ def self.csv(patients)
           :phone_numbers,
           :address,
           :medical_history,
-          :prescription_drugs,
+          :current_prescription_drugs,
           :latest_bp_passports,
           { latest_scheduled_appointments: :facility },
-          { latest_blood_pressures: :facility },
+          { latest_blood_pressures: [:facility, {encounter: :blood_sugars}] },
```

Comment:
> Populates blood sugar data into the new columns if and only if the patient had a blood sugar reading on the same day as their latest BP

Is there a reason to not add patients who have just blood sugars? Sorry a bit out of the loop on the discussion around this.

Reply:
Totally. We've established that _most_ of our patients will definitely have a BP along with their blood sugar reading: https://simpledotorg.slack.com/archives/CFJMFSM8F/p1592304255188700

So this PR is just meant to _enrich_ the hypertension line list.